impeller/entity/geometry/geometry.h

```diff
@@ -62,22 +62,24 @@ class Geometry {
 
   static std::shared_ptr<Geometry> MakeCover();
 
-  static std::shared_ptr<Geometry> MakeRect(Rect rect);
+  static std::shared_ptr<Geometry> MakeRect(const Rect& rect);
 
-  static std::shared_ptr<Geometry> MakeLine(Point p0,
-                                            Point p1,
+  static std::shared_ptr<Geometry> MakeOval(const Rect& rect);
+
+  static std::shared_ptr<Geometry> MakeLine(const Point& p0,
+                                            const Point& p1,
                                             Scalar width,
                                             Cap cap);
 
-  static std::shared_ptr<Geometry> MakeCircle(Point center, Scalar radius);
+  static std::shared_ptr<Geometry> MakeCircle(const Point& center,
+                                              Scalar radius);
 
-  static std::shared_ptr<Geometry> MakeStrokedCircle(Point center,
+  static std::shared_ptr<Geometry> MakeStrokedCircle(const Point& center,
                                                      Scalar radius,
                                                      Scalar stroke_width);
 
-  static std::shared_ptr<Geometry> MakePointField(std::vector<Point> points,
-                                                  Scalar radius,
-                                                  bool round);
+  static std::shared_ptr<Geometry>
+  MakePointField(const std::vector<Point>& points, Scalar radius, bool round);
```

Comment:
Same here, we don't enforce it but I believe we only std::move into this constructor